src/common/tests/umem_test_bmem.c

```diff
@@ -213,14 +213,17 @@ setup_pmem_internal(void **state, struct umem_store *store)
 	struct test_arg		*arg = *state;
 	static int		 tnum;
 	int			 rc = 0;
+	uint64_t                 pool_sz = POOL_SIZE;
 
 	D_ASPRINTF(arg->ta_pool_name, "/mnt/daos/umem-test-%d", tnum++);
 	if (arg->ta_pool_name == NULL) {
 		print_message("Failed to allocate test struct\n");
 		return 1;
 	}
 
-	rc = utest_pmem_create(arg->ta_pool_name, POOL_SIZE * 2, sizeof(*arg->ta_root), store,
+	if (store->store_type == DAOS_MD_BMEM_V2)
+		pool_sz = POOL_SIZE * 2;
```

Comment:
is this just hardcoded for unit tests, I'm presuming there are other tests that variable mem-ratio? Just checking.